config/initializers/file_set_derivatives_overrides.rb

```diff
@@ -1,5 +1,8 @@
 # This file contains overrides to Hyrax::FileSetDerivativesService to increase the size of thumbnails
 
+# @see https://github.com/samvera/hydra-derivatives/blob/main/lib/hydra/derivatives/processors/video/config.rb#L59
+DEFAULT_VIDEO_SIZE = '320x240'
```

Comment:
@labradford it looks like this constant is defined outside of any class, making it "global" (i.e. `Object::DEFAULT_VIDEO_SIZE` instead of `Hyrax::...::DEFAULT_VIDEO_SIZE`). Does it have to be?

Reply:
Checked with Kirk and this was not intentional. I moved it inside the class